docker-compose.dev.yml

```diff
@@ -53,6 +56,7 @@ services:
             service: kafka
         ports:
             - '9092:9092'
+        restart: on-failure
```

Comment:
Looks like we already have restart on failure set in the base docker compose from which we extend, does this not carry over?

Reply:
Weird, that didn't seem like the case for me before, but maybe I overlooked the restarts 🤔 Reverted this change